ide-common/src/main/java/org/digma/intellij/plugin/analytics/AnalyticsService.java

```diff
@@ -108,6 +108,7 @@ public AnalyticsService(@NotNull Project project) {
             if (shouldReplaceClient) {
                 Log.log(LOGGER::debug, "api url changed to {}, calling replace client", myApiUrl);
                 replaceClient(myApiUrl);
+                AuthManager.getInstance().logout();
```

Comment:
looks strange to logout after replacing the client because replaceClient will do a login and then logout after that will delete the account